src/main.c

```diff
@@ -46,6 +47,24 @@ static void user_configuration(bool run)
 	set_speed_mode(mode, run);
 }
 
+/**
+ * @brief Check battery voltage and warn if the voltage is getting too low.
+ */
+static void check_battery_voltage(void)
+{
+	float voltage;
+
+	voltage = get_battery_voltage();
+	if (voltage < 3.6)
+		speaker_warn_low_battery();
```

Comment:
Why not only one if with conditions:  (voltage < 3.6) && (voltage > 3.2)? why not else if?

Reply:
To chain "low battery" sounds. Would give an idea on how low the battery is.

- 2 sounds for < 3.6
- 4 sounds for < 3.5
- ...
- Many sounds for < 3.3 plus a long sound for "critically low".